sdk/src/test/scala/io/horizen/account/api/rpc/service/EthServiceTest.scala

```diff
@@ -910,7 +915,12 @@ class EthServiceTest extends JUnitSuite with MockitoSugar with ReceiptFixture wi
       ("0x1234567890123456789012345678901234567890", "latest", "\"0x\"")
     )
 
-    val invalidCases = Table(("Address", "Tag"), ("0x", "latest"), ("0x1234567890123456789012345678901234567890", ""))
+    val invalidCases = Table(
+      ("Address", "Tag"),
+      ("0x", "latest"),
+      ("0x1234567890123456789012345678901234567890", ""),
+      ("0x1234567891011121314151617181920212223242", "0x1337")
```

Comment:
let's add comments for each case explaining why it is considered to be invalid.
Same in other places.